src/docs/concepts-payments/payments-overview/index.md

```diff
@@ -264,6 +264,14 @@ listings or purchase preorder products further in the future than the 90
 day Stripe limitation, and they will be charged closer to the moment of
 receiving the product or service they purchased.
 
+### Negotiated payments
+
+One of the default processes in Sharetribe involves a negotiated
+payment. In this process, the listing price does not determine the price
+of the transaction, and instead the customer and provider negotiate the
+price between them. Once both the provider and customer have accepted an
+offer, the customer proceeds to make a payment based on that price.
+
```

Comment:
Seems vague to say "one of the default processes"; why not say "The negotiation process, which is one of the default processes,"